auth-api/src/auth_api/services/user.py

```diff
@@ -71,7 +73,7 @@ def as_dict(self):
 
     @staticmethod
     def create_user_and_add_membership(memberships: List[dict], org_id, token_info: Dict = None,
```

Comment:
Please add some code comments on what the logic is; as this method is large now

Reply:
ok..Noted..Will be in next PR..